misk/src/main/java/misk/ServiceBindings.java

```diff
@@ -0,0 +1,12 @@
+package misk;
+
+import com.google.common.util.concurrent.Service;
+import com.google.inject.Key;
+import com.google.inject.TypeLiteral;
+import java.util.Set;
+
+final class ServiceBindings {
```

Comment:
why Java?

Reply:
I couldn't figure out the required magic incantations to produce the proper `TypeLiteral` in Kotlin (`@JvmSuppressWildcards` didn't do it)